apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component, PropTypes} from 'react';
+
+class ManageStudentAgeCell extends Component {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    age: PropTypes.number,
+    isEditing: PropTypes.bool,
+  };
+
+  state = {
+    ageValue: this.props.age
+  };
+
+  onChangeAge = (e) => {
+    this.setState({ageValue: e.target.value});
+  }
+
+  render() {
+    const {age} = this.props;
+    const ages = ['', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14',
+      '15', '16', '17', '18', '19', '20', '21+'];
+    return (
+      <div>
+        {
+          !this.props.isEditing &&
+          <div>
+            {age}
+          </div>
+        }
+        {
+          this.props.isEditing &&
+          <select
+            ref={element => this.root = element}
+            name="age"
+            value={this.state.ageValue}
+            onChange={this.onChangeAge}
+          >
+           {ages.map(age => <option key={age} value={age}>{age}</option>)}
+          </select>
+        }
+      </div>
+    );
+  }
+}
+
+export default ManageStudentAgeCell;
```

Comment:
Can/should we share functionality with https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/AgeDropdown.jsx

Reply:
I've shared the list of possible ages, but as discussed offline AgeDropdown isn't very "react-y" in how it handles it's state, so for now I'm going to keep it separate, and I'll come back to it once I'm using actual data/redux to see what works best.